.travis.yml

```diff
@@ -20,7 +19,8 @@ matrix:
       env: PHPSTAN=1
 
 before_script:
-  - composer --prefer-source install
+  - if [[ $TRAVIS_PHP_VERSION = hhvm ]]; then composer --ignore-platform-reqs --prefer-source install; fi
```

Comment:
Fine, I tried. 😊 Damn I hate HHVM for this versioning hell... When switched to php7 mode (`hhvm --php -d hhvm.php7.all=true`), it works, but it's messy...